### DISCUSSION TITLE
'Post build' Hook

### DISCUSSION THREAD

**Discussion by alexcibotari** [analogjs/analog#1335] [Ideas]
### Which scope/s are relevant/related to the feature request?

vite-plugin-angular

### Information

As a developer, I need a hook at the end of the build when all pages are rendered, and I can access them to create the index for search. At the moment Analog exposes ``postRenderingHooks`` after each page render. this is not enough to create a site search index, like a sitemap, where content for each page will be also available. or at least a hook with all generated paths, and I can traverse by my self.

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [ ] No

**Comment by brandonroberts** [comment]
There's a hook in `nitro` for this

```ts
analog({
  nitro: {
    hooks: {
      "prerender:done": ({ prerenderedRoutes }) => {
        console.log(prerenderedRoutes);
      }
    }
  }
})
```

**Comment by alexcibotari** [comment]
The issue is, that they will not run during: `npm run dev`, event after build `npm run dev` will delete the file
